AutolayoutPractice/RemindersList/Views/AddReminderViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AddReminderViewController.swift
+//  AutolayoutPractice
+//
+//  Created by Amanda Benavente on 6/23/24.
+//
+
+import UIKit
+
+protocol AddReminderDelegate: AnyObject {
+    func addReminder(title: String, description: String, dueDate: Date)
+}
+
+class AddReminderViewController: UIViewController {
+
+    weak var delegate: AddReminderDelegate?
+
+    private let navBar = UINavigationBar()
+
+    private let titleTextField: UITextField = {
+        let textField = UITextField()
+        textField.borderStyle = .roundedRect
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        textField.placeholder = "Label for your reminder"
+        return textField
+    }()
+
+    private let descriptionLabel: UILabel = {
+        let lbl = UILabel()
+        lbl.translatesAutoresizingMaskIntoConstraints = false
+        lbl.text = "Add details for your reminder"
+        return lbl
+    }()
+
+    private let descriptionTextView: UITextView = {
+        let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.layer.cornerRadius = 5
+        textView.layer.borderColor = UIColor.lightGray.withAlphaComponent(0.4).cgColor
+        textView.layer.borderWidth = 1
+        textView.font = .preferredFont(forTextStyle: .body)
+        return textView
+    }()
+
+    private let dueDatePicker: UIDatePicker = {
+        let datePicker = UIDatePicker()
+        datePicker.translatesAutoresizingMaskIntoConstraints = false
+        datePicker.preferredDatePickerStyle = .inline
+        datePicker.minimumDate = Date()
+        return datePicker
+    }()
+
+    private let notificationPicker: UIPickerView = {
+        UIPickerView()
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemBackground
+        configure()
+    }
+
+    private func configureNavBar() {
+        navBar.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: 50)
```

Comment:
How come we're using a frame here? Are you running into any autolayout constraint warnings in the debugger?